concent_api/core/exceptions.py

```diff
@@ -9,6 +9,10 @@ class Http400(ConcentBaseException):
     pass
 
 
+class Http500(ConcentBaseException):
```

Comment:
Shouldn't we name exceptions more specific? This name is too general. If there exist need to add more exceptions just do it :)